run.py

```diff
@@ -260,13 +233,20 @@ def main(gtk_context):
             e = "gear-dry-run is set: Command was NOT run."
             log.warning(e)
             warnings.append(e)
-            pretend_it_ran(gtk_context)
+            pretend_it_ran(destination_id)
 
         else:
             # Create output directory
             log.info("Creating output directory %s", output_analysis_id_dir)
             Path(output_analysis_id_dir).mkdir()
 
+            if config["gear-log-level"] != "INFO":
```

Comment:
Good point.  Right now the manifest only allows INFO or DEBUG.  So INFO is kind of like "quiet".  If other logging levels would add more logging, this would still be appropriate.  But if less logging is desired, this will have to be changed.